problems/2017-04-11-side-effects-may-include/tests/getYearDuration.spec.js

```diff
@@ -0,0 +1,11 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+
+const getYearDuration = require('../lib/getYearDuration').default;
+
+describe('getYearDuration', () => {
```

Comment:
I like this example since it shows a nice function that is pure and you can kinda tell easily at a glance. I kind of think the test is somewhat confusing because I don't quite know if I'm supposed to fix something about it or not.

Maybe an additional comment in here reminding the me's of the world out there that this thing is OK would help?